t/lib/Test/Code/TidyAll/Basic.pm

```diff
@@ -437,6 +437,21 @@ sub test_errors : Tests {
     throws_ok { $ct->process_paths("$other_dir/foo.txt") } qr/not underneath root dir/;
 }
 
+sub test_diff_on_tidy_error : Tests {
```

Comment:
Won't this fail on any system that doesn't have diff installed?  Can we skip somehow?